frontend/src/pages/SettingsPage/SettingsPage.tsx

```diff
@@ -46,6 +46,7 @@ export const SettingsPage = () => {
 
   useEffect(() => {
     analytics.page('SettingsPage')
+    emit('interfaces')
```

Comment:
remove

Reply:
I kept this one.  I'd like to have it on the connections page, devices page and settings page. So we have a better chance of it getting updated when a user comes back to the app.  If you have another idea for this let me know.